dev/ScrollViewer/ScrollViewer_rs1_themeresources.xaml

```diff
@@ -0,0 +1,43 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+    <ResourceDictionary.ThemeDictionaries>
+        <ResourceDictionary x:Key="Default">
+            <SolidColorBrush x:Key="ScrollViewerScrollBarsSeparatorBackground" Color="{StaticResource SystemChromeMediumColor}" Opacity="0.9" />
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorExpandDelay">00:00:00</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorExpandDuration">00:00:00.1</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorContractDelay">00:00:02</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorContractDelayDisabled">00:00:00</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorContractDuration">00:00:00.1</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorDisplayDelayWithoutAnimation">00:00:00.725</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorExpandDelayWithoutAnimation">00:00:00</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorContractDelayWithoutAnimation">00:00:02</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsNoTouchDuration">00:00:00.5</x:String>
+        </ResourceDictionary>
+        <ResourceDictionary x:Key="Light">
+            <SolidColorBrush x:Key="ScrollViewerScrollBarsSeparatorBackground" Color="{StaticResource SystemChromeMediumColor}" Opacity="0.9" />
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorExpandDelay">00:00:00</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorExpandDuration">00:00:00.1</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorContractDelay">00:00:02</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorContractDelayDisabled">00:00:00</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorContractDuration">00:00:00.1</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorDisplayDelayWithoutAnimation">00:00:00.725</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorExpandDelayWithoutAnimation">00:00:00</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsSeparatorContractDelayWithoutAnimation">00:00:02</x:String>
+            <x:String x:Key="ScrollViewerScrollBarsNoTouchDuration">00:00:00.5</x:String>
+        </ResourceDictionary>
+        <ResourceDictionary x:Key="HighContrast">
```

Comment:
These values don't seem different between the themes. Why do you have them broken out?

Reply:
Ah, good point. I fixed it by copying the simplified TabView pattern. Thanks.